packages/expo-updates/e2e/setup/project.ts

```diff
@@ -221,12 +211,20 @@ async function preparePackageJson(
 
   const extraScriptsGenerateTestUpdateBundlesPart = shouldGenerateTestUpdateBundles
     ? {
-        'generate-test-update-bundles': 'node scripts/generate-test-update-bundles.js',
+        'generate-test-update-bundles': 'npx ts-node ./scripts/generate-test-update-bundles',
       }
     : {
         'generate-test-update-bundles': 'echo 1',
       };
 
+  const extraScriptsAssetExclusion = {
+    'reset-to-embedded': 'npx ts-node ./scripts/reset-app.ts App.tsx.embedded',
+    'set-to-update-1':
+      'npx ts-node ./scripts/reset-app.ts App.tsx.update1; eas update --branch=main --message=Update1',
+    'set-to-update-2':
+      'npx ts-node ./scripts/reset-app.ts App.tsx.update2; eas update --branch=main --message=Update2',
+  };
```

Comment:
optional: might be faster to use `yarn --silent ts-node --transpile-only` instead of `npx ts-node` since ts-node is installed locally by the dev dependency you added.